tests/data.py

```diff
@@ -652,6 +649,56 @@ class Galadriel:
     }
 
 
+class GaladrielDeviceID:
+    """Test data of the galadriel platform."""
+
+    name = "galadriel"
+
+    device_id = 9
+
+    gates_settings: dict[str, Any] = {
+        PLATFORM.MINIMUM_CLOCK_TIME: 4,
+        PLATFORM.DELAY_BETWEEN_PULSES: 0,
+        PLATFORM.DELAY_BEFORE_READOUT: 0,
+        PLATFORM.TIMINGS_CALCULATION_METHOD: "as_soon_as_possible",
+        PLATFORM.RESET_METHOD: ResetMethod.PASSIVE.value,
+        PLATFORM.PASSIVE_RESET_DURATION: 100,
+        "operations": [],
+        "gates": {},
+    }
+
+    instruments: list[dict] = []
+    instrument_controllers: list[dict] = []
+
+    chip: dict[str, Any] = {
+        "nodes": [],
+    }
+
+    buses: list[dict[str, Any]] = []
+
+    runcard: dict[str, Any] = {
+        RUNCARD.NAME: name,
+        RUNCARD.DEVICE_ID: device_id,
```

Comment:
I would write it like this, so we can remove it from the `constants.py` already, and don't forget in the future, and also it makes it easier to spot maybe? (well that not important really hehe)
```suggestion
        "device_id": device_id,
```

Reply:
Done!